static/index.html

```diff
@@ -10,25 +10,37 @@
     <meta property="og:title" content="cloud.gov deck">
     <meta property="og:url" content="https://console.cloud.gov/">
 
+    <script>
+      if (window.location.host === 'dashboard.cloud.gov') {
+        window.PRODUCTION = 1;
```

Comment:
I'm chill with this since as long as it is defined it will serve the purpose, but I would prefer this to be a boolean of `true`